projects/addon-editor/tokens/editor-enable-image-preview.ts

```diff
@@ -0,0 +1,6 @@
+import {InjectionToken} from '@angular/core';
+
+export const TUI_EDITOR_ENABLE_IMAGE_PREVIEW = new InjectionToken<boolean>(
```

Comment:
Perhaps we need to add information about the token in the `DI tokens` tab

Reply:
fixed